packages/selenium/test/RunPartial/IncludeExcludeTests.cs

```diff
@@ -31,11 +46,18 @@ public void ShouldHonorExclude(string browser)
 
             var axe = new AxeBuilder(WebDriver)
                 .Exclude(".exclude");
-            var excludeRes = axe.Analyze();
+            var exclude1Res = axe.Analyze();
+
+            var axe2 = new AxeBuilder(WebDriver)
+                .Exclude(".exclude")
+                .Exclude(".exclude2");
+            var exclude2Res = axe2.Analyze();
 
-            var axe2 = new AxeBuilder(WebDriver);
-            var normalRes = axe2.Analyze();
-            Assert.That(excludeRes.Violations.Length, Is.LessThan(normalRes.Violations.Length));
+            var targets2 = getPassTargets(exclude2Res);
+            var targets1 = getPassTargets(exclude1Res);
+            Assert.IsNull(targets1.Find(t => t.Equals(".exclude")));
+            Assert.IsNull(targets2.Find(t => t.Equals(".exclude")));
+            Assert.IsNull(targets2.Find(t => t.Equals(".exclude2")));
```

Comment:
Missing assertion here that `targets1` still includes `.exclude2` (similar to the above)

Reply:
If we go by the [exclude test of axe test fixtures](https://github.com/dequelabs/axe-test-fixtures/blob/v1/README.md?plain=1#L334-L343) then just checking that the targets don't include what we expect is fine